tf-build-tools/tf-maven-plugin/pom.xml

```diff
@@ -15,6 +15,7 @@
     <maven.install.skip>false</maven.install.skip>
     <gpg.skip>false</gpg.skip>
     <skipNexusStagingDeployMojo>false</skipNexusStagingDeployMojo>
+    <excludedGroups>com.deliveredtechnologies.test.categories.Integration</excludedGroups>
```

Comment:
This excludes Integration test group(s) by default.
The Integration test group includes tests that require external bootstrapping.